xbmc/network/Socket.cpp

```diff
@@ -81,7 +82,8 @@ bool CPosixUDPSocket::Bind(bool localOnly, int port, int range)
           m_iSock = INVALID_SOCKET;
         }
 
-        closesocket(testSocket);
+        if (testSocket != INVALID_SOCKET)
```

Comment:
this code will never run

Reply:
How comes?
This code is definitely run otherwise it wouldn't fix anything.
I saw it in the debugger on my system!

Reply:
I confused `testSocket` and `m_iSock` - disregard :-)

Reply:
lulz I was still right xD